src/app/app-routing.module.ts

```diff
@@ -42,9 +44,19 @@ const routes: Routes = [
       },
     ],
   },
+  {
+    path: '',
+    component: WithloginunverifiedlayoutComponent,
+    canActivate: [AuthGuard],
+    children: [
+      // Other routes available only to logged-in but unverified users
+    ],
+  },
   {
     path: '',
     component: WithloginlayoutComponent,
+    //auther authentication guards will be added here
+    canActivate: [AuthGuard],
```

Comment:
can you create a guard which would be for `AuthVerifiedGuard` so users are both logged in and verified. it would still inside it use the Auth Guard but perform extra checkes